apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-5_6/pom.xml

```diff
@@ -32,9 +32,9 @@
 
         <!-- Elasticsearch testcontainer -->
         <dependency>
-            <groupId>fr.pilato.elasticsearch.testcontainers</groupId>
-            <artifactId>testcontainers-elasticsearch</artifactId>
-            <version>0.1</version>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>elasticsearch</artifactId>
+            <version>${version.testcontainers}</version>
```

Comment:
FYI I would highly recommend to use our BOM :)
```org.testcontainers:testcontainers-bom:1.10.1`

/cc @felixbarny

Reply:
Thx for the tips. PR is here: https://github.com/elastic/apm-agent-java/pull/295